src/main/java/org/generationcp/middleware/api/location/LocationService.java

```diff
@@ -17,27 +16,22 @@
 	 *
 	 * @param locationSearchRequest - filter parameters
 	 * @param pageable              - pagination parameters
+	 * @param programUUID
 	 * @return
 	 */
-	List<Location> getFilteredLocations(LocationSearchRequest locationSearchRequest, Pageable pageable);
+	List<LocationDTO> searchLocations(LocationSearchRequest locationSearchRequest, Pageable pageable,
+			final String programUUID);
 
 	/**
 	 * Returns the count of Location records filtered by LocationSearchRequest parameter.
 	 *
 	 * @param locationSearchRequest - filter parameters
-	 * @return
-	 */
-	long countFilteredLocations(LocationSearchRequest locationSearchRequest);
-
-	/**
-	 * Gets the favorite project location ids.
-	 *
-	 * @param programUUID - unique id of program
-	 * @return the favorite project location ids
+	 * @param programUUID
+   * @return
 	 */
-	List<Integer> getFavoriteProjectLocationIds(String programUUID);
+	long countFilteredLocations(LocationSearchRequest locationSearchRequest, final String programUUID);
 
-	List<org.generationcp.middleware.api.location.Location> getLocations(LocationSearchRequest locationSearchRequest, Pageable pageable);
+  List<org.generationcp.middleware.api.location.Location> getLocations(LocationSearchRequest locationSearchRequest, Pageable pageable);
```

Comment:
@lik-83 seems the file is not properly formatted. Could you please check?

Reply:
Done